packages/webdriverjs/package.json

```diff
@@ -57,32 +57,32 @@
     "webdriverjs"
   ],
   "devDependencies": {
-    "@types/chai": "^4.3.1",
+    "@types/chai": "^4.3.3",
     "@types/chromedriver": "^81.0.1",
-    "@types/express": "^4.17.13",
+    "@types/express": "^4.17.14",
     "@types/http-server": "^0.12.1",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "^18.0.3",
-    "@types/selenium-webdriver": "^4.1.1",
-    "@types/sinon": "^10.0.12",
+    "@types/mocha": "^10.0.0",
+    "@types/node": "^18.8.3",
+    "@types/selenium-webdriver": "^4.1.5",
+    "@types/sinon": "^10.0.13",
     "@types/test-listen": "^1.1.0",
-    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures#v1",
+    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures",
     "chai": "^4.3.6",
     "chaimocha": "^1.10.0",
-    "chromedriver": "^105.0.1",
+    "chromedriver": "^106.0.1",
     "delay": "^5.0.0",
-    "express": "^4.17.1",
+    "express": "^4.18.2",
     "expresss": "^0.0.0",
     "http-server": "^14.1.1",
     "is-ci": "^3.0.1",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0",
     "rimraf": "^3.0.2",
-    "selenium-webdriver": "^4.3.0",
-    "sinon": "^14.0.0",
+    "selenium-webdriver": "4.3.0",
```

Comment:
Will need to pin at `4.3.0` - 

https://github.com/dequelabs/axe-core-npm/issues/580 - breaking change introduced